evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -690,37 +691,40 @@ def __init__(self, random_arg=False, numpy_arg=[], random_seed=0):
         parameters={"My Custom Estimator": {"random_arg": Imputer()}},
     )
     generated_pipeline_code = generate_pipeline_code(pipeline)
-    assert (
-        generated_pipeline_code
-        == "from evalml.pipelines.binary_classification_pipeline import BinaryClassificationPipeline\n"
+    assert generated_pipeline_code == black.format_str(
+        "from evalml.pipelines.binary_classification_pipeline import BinaryClassificationPipeline\n"
         "pipeline = BinaryClassificationPipeline(component_graph={'Imputer': ['Imputer', 'X', 'y'], 'My Custom Estimator': [CustomEstimator, 'Imputer.x', 'y']}, "
         "parameters={'Imputer':{'categorical_impute_strategy': 'most_frequent', 'numeric_impute_strategy': 'mean', 'boolean_impute_strategy': 'most_frequent', 'categorical_fill_value': None, 'numeric_fill_value': None, 'boolean_fill_value': None}, "
         "'My Custom Estimator':{'random_arg': Imputer(categorical_impute_strategy='most_frequent', numeric_impute_strategy='mean', boolean_impute_strategy='most_frequent', categorical_fill_value=None, numeric_fill_value=None, boolean_fill_value=None), 'numpy_arg': []}}, "
-        "custom_name='Mock Binary Pipeline with Transformer', random_seed=0)"
+        "custom_name='Mock Binary Pipeline with Transformer', random_seed=0)",
+        mode=black.Mode(target_versions={black.TargetVersion.PY39}, line_length=88),
     )
 
 
 def test_generate_code_pipeline():
     binary_pipeline = BinaryClassificationPipeline(
         ["Imputer", "Random Forest Classifier"],
     )
-    expected_code = (
+    expected_code = black.format_str(
         "from evalml.pipelines.binary_classification_pipeline import BinaryClassificationPipeline\n"
         "pipeline = BinaryClassificationPipeline(component_graph={'Imputer': ['Imputer', 'X', 'y'], 'Random Forest Classifier': ['Random Forest Classifier', 'Imputer.x', 'y']}, "
         "parameters={'Imputer':{'categorical_impute_strategy': 'most_frequent', 'numeric_impute_strategy': 'mean', 'boolean_impute_strategy': 'most_frequent', 'categorical_fill_value': None, 'numeric_fill_value': None, 'boolean_fill_value': None}, "
-        "'Random Forest Classifier':{'n_estimators': 100, 'max_depth': 6, 'n_jobs': -1}}, random_seed=0)"
+        "'Random Forest Classifier':{'n_estimators': 100, 'max_depth': 6, 'n_jobs': -1}}, random_seed=0)",
+        mode=black.Mode(target_versions={black.TargetVersion.PY39}, line_length=88),
```

Comment:
Is `line_length=88` arbitrary? Wondering if it's worth making it a param or global variable.

Reply:
It's what we have in our black config. Let me see how I can clean it up!